sdk/monitor/azure-monitor-opentelemetry-exporter/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 ### Bugs Fixed
 - Fix for issue #41470. Added back exception message, removed earlier due to linting errors
   ([#41512] https://github.com/Azure/azure-sdk-for-python/pull/41512)
+- Should not populate device OS tag in the exporter.
```

Comment:
NIT: could leave a more detailed comment like "leave ingestion to populate Device OS from User Agent"

Reply:
Updated